src/main/java/chess/GameController.java

```diff
@@ -0,0 +1,57 @@
+package chess;
+
+import chess.domain.board.Board;
+import chess.domain.location.Location;
+import chess.view.InputView;
+import chess.view.OutputView;
+
+public class GameController {
+    private static final InputView inputView = new InputView();
+    private static final OutputView outputView = new OutputView();
```

Comment:
```suggestion
    private static final InputView INPUT_VIEW = new InputView();
    private static final OutputView OUTPUT_VIEW = new OutputView();
```
static 변수는 대문자 snake case를 사용해주세요